package.json

```diff
@@ -144,8 +144,13 @@
   },
   "pnpm": {
     "peerDependencyRules": {
-      "allowAny": ["eslint"],
-      "ignoreMissing": ["@babel/core", "postcss"]
+      "allowAny": [
+        "eslint"
+      ],
+      "ignoreMissing": [
+        "@babel/core",
+        "postcss"
+      ]
```

Comment:
Did you accidentally auto format this?

Reply:
I never opened this file in my editor, so it would have been pnpm to have done this

Reply:
I ran `pnpm up -iL`